suite-common/wallet-utils/src/accountUtils.ts

```diff
@@ -246,15 +263,47 @@ export const getAccountTypeTech = (path: string) => {
     return 'TR_ACCOUNT_TYPE_BIP44_TECH';
 };
 
-export const getAccountTypeDesc = (path: string) => {
+type getAccountTypeDescProps = {
```

Comment:
there is 4 params here, but `getAccountTypeDesc` accepts just 3

Reply:
Hm, maybe I didn't push some changes at that point, now it's 3